t/19-isotovideo-command-processing.t

```diff
@@ -394,6 +394,34 @@ subtest signalhandler => sub {
     } qr/isotovideo received signal INT/, 'Signal logged';
     is($last_signal, 'INT', 'Event emitted');
 };
+subtest 'Check exit_code_from_test_results' => sub {
+    my $runner = OpenQA::Isotovideo::Runner->new;
+    use Mojo::File 'path';
+    use Mojo::JSON 'encode_json';
+    my %fake_results = ();
```

Comment:
```suggestion
    my %fake_results;
```

But probably it makes sense to remove the variable completely, see my other suggestion.

Reply:
sure. i removed it